asciinema/tests/test_config.py

```diff
@@ -0,0 +1,212 @@
+import unittest
```

Comment:
This file was renamed and changed as well, so the diff is not representative unfortunately, it would be better to compare the current `tests/config_test.py` with it for a proper diff.